options.go

```diff
@@ -2,31 +2,22 @@ package gosundheit
 
 type Option func(*health)
 
-// WithCheckListener allows you to listen to check start/end events
-func WithCheckListener(listener CheckListener) Option {
+// WithCheckListeners allows you to listen to check start/end events
+func WithCheckListeners(listener ...CheckListener) Option {
```

Comment:
nit: I'd call it `WithCheckListener` regardless the fact that it accepts multiple listeners.

Reply:
It was the original method name and then @eranharel asked to use plural